src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/PrisonerServiceTest.kt

```diff
@@ -53,12 +53,12 @@ class PrisonerServiceTest {
       givenName = "John",
       familyName = "Doe",
       dateOfBirth = prisonerDOB,
-      otherIdentifiers = OtherIdentifiers(pncIdentifier = PNCIdentifier("PNC123")),
+      otherIdentifiers = OtherIdentifiers(pncIdentifier = PNCIdentifier("PNC123"), prisonNumber = "12345"),
```

Comment:
I know it is not the focus of this change, but it is a shame we still have so many invalid PNCs knocking about in tests